src/main/kotlin/org/rust/cargo/commands/Cargo.kt

```diff
@@ -70,13 +72,16 @@ class Cargo(
 
     private val metadataCommandline: GeneralCommandLine get() = generalCommand("metadata", listOf("--verbose"))
 
+    private fun rustfmtCommandline(filePath: String) =
+        generalCommand("fmt").withParameters("-v", "--", "--write-mode=overwrite", "--skip-children", filePath)
```

Comment:
the verbose flag will just output addition message whenever the reformat action success or fail like 

```
[Bin] "/Users/zjh/Documents/rustspace/hello_world/src/main.rs"
rustfmt --write-mode=display --skip-children foo.rs /Users/zjh/Documents/rustspace/hello_world/src/main.rs
```

so maybe a little nonsense here, i'll remove that. i used to inspect reformat procedure use this flag.